src/main/java/org/myrobotlab/service/interfaces/ServiceLifeCycle.java

```diff
@@ -0,0 +1,17 @@
+package org.myrobotlab.service.interfaces;
+
+import org.myrobotlab.framework.Registration;
+
+public interface ServiceLifeCycle {
+  
+  public Registration registered(Registration registration);
+
+  public String created(String serviceName);
```

Comment:
If this is actually a publishing sort of interface. it should be publishCreated / publishStarted ... etc..  and these should just be pushed down to the Service class for their implementation.  Generally these shouldn't be overridden in subclasses, and this should be called ServiceLifeCyclePublisher ...

Reply:
It always has - its legacy that it wasn't publishRegistered.
It can be done in a later pr.